operations/app/terraform/modules/function_app/locals.tf

```diff
@@ -12,6 +12,9 @@ locals {
     "OKTA_redirect" = var.okta_redirect_url
     "OKTA_authkey"  = var.OKTA_authKey
     "OKTA_ClientId" = var.OKTA_clientId
+    "RS_OKTA_redirect" = var.RS_okta_redirect_url
```

Comment:
missing `"RS_OKTA_baseUrl"  = var.RS_okta_base_url`?

Reply:
Added RS_OKTA_baseUrl in the latest commit